src/main/java/de/tum/in/www1/artemis/domain/exam/Exam.java

```diff
@@ -165,12 +165,12 @@ public void setTestExam(boolean testExam) {
         this.testExam = testExam;
     }
 
-    public boolean isMonitoring() {
-        return monitoring;
+    public boolean isLiveStatistics() {
```

Comment:
Maybe `isLiveStatisticsEnabled`? (Could also rename attribute.)

Reply:
I would stick to the previous naming, so i leave it as it is :+1: